### PR TITLE
eval-cache tests rework

### PR DESCRIPTION
**Pull request by tomasz-biernacki**
Rename the files during tests in order to leave the git repo intact

### CODE REVIEW

File: test/node/replumb/cache_node_test.cljs [hunk 1]
```diff
@@ -9,6 +9,18 @@
             [replumb.load :as load]
             [replumb.nodejs.io :as io]))
 
+(defn rename-file-names-to-tmp
```
Comment (arichiardi) [inline test/node/replumb/cache_node_test.cljs]: Maybe `move` is a more suitable name?

Comment (tomasz-biernacki) [inline test/node/replumb/cache_node_test.cljs]: I would leave `rename` since this is what it does

Comment (arichiardi) [inline test/node/replumb/cache_node_test.cljs]: well, in unix actually a rename is always a `move` but ok :smile:

Comment (tomasz-biernacki) [inline test/node/replumb/cache_node_test.cljs]: yes, you are right, I will _rename_ it to `move` :dancer:

Comment (arichiardi) [inline test/node/replumb/cache_node_test.cljs]: (- :lollipop: "lipop") but no, don't worry

**Comment by arichiardi** [comment]
Looks good to me, shall I merge?

**Comment by tomasz-biernacki** [comment]
Let's try and see what happens ;)

**Pull request merged**